packages/panda-san/src/navbar/navbar.scss

```diff
@@ -0,0 +1,107 @@
+.navbar {
+  align-items: center;
+  background-color: $navbar;
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  left: 0;
+  position: absolute;
+  width: 100vw;
+  z-index: 100;
+
+  &__brand {
+    display: flex;
+    flex: 1;
+
+    &__icon {
+      cursor: pointer;
```

Comment:
I don't think it should be by default `pointer`. This would be up to the `a` or `button` element, if there was one.